README.md

```diff
@@ -121,6 +121,18 @@ Response will typically have 201 status code and the following details:
 
 ```
 
+Possible error codes are: 
+
+|Error Code                              | Status Code | Meaning                                                                       |
+|----------------------------------------|-------------|-------------------------------------------------------------------------------|
+|slack_error                             | 4xx/5xx     | A generic error occurring when the service is unable to notify a slack channel|
```

Comment:
The table is still unclear to me. I suggest you remove the status from there and define what errors can be found in the payload case the API returns successfully.

After the table an additional paragraph could what errors can occur when the API returns something different than 200.
4xx/5xx is quite a wide range of errors, I don't think they should be grouped together.
4xx should be used when the caller of the API made an error, not when there was a problem contacting slack. For example 400 should be returned if the payload sent by the caller of this API was incorrect. In my opinion any error occurred when talking to Slack should result in a 500 (not 5xx). The service should never return any other 5xx status